yorkie/src/main/kotlin/dev/yorkie/core/Client.kt

```diff
@@ -182,7 +182,7 @@ public class Client @VisibleForTesting internal constructor(
     }
 
     private fun filterRealTimeSyncNeeded() = attachments.value.filterValues { attachment ->
-        attachment.syncMode.needRealTimeSync &&
+        attachment.needRealTimeSync() &&
             (attachment.document.hasLocalChanges || attachment.remoteChangeEventReceived)
```

Comment:
how about moving all the logic to `Attachment.needRealTimeSync`?

Reply:
that's much better, I think. done in [6b7ebaf](https://github.com/yorkie-team/yorkie-android-sdk/pull/188/commits/6b7ebaf66d8b713200248796e6eba71bdb7f4836)